RELEASE-NOTES.md

```diff
@@ -105,6 +110,15 @@ mkdir /tmp/node/db
 
 #### Start node
 
+It is recommended that at least 1024 files can be opened in the session where you start the node: this is needed not only for files on disk (e.g. persisted chain) but also for network connections with peers.
```

Comment:
A higher number of file descriptor may be harder to set because of other configs to change.

Reply:
Indeed:
```
$ ulimit -n 1025
-bash: ulimit: open files: cannot modify limit: Operation not permitted
```

Reply:
But this is once you change it to 1024, if you don't have limit set yet, you can set it up to kern.maxfilesperproc

Reply:
The limits looks even higher on linux.
So, I think it's pretty safe to have higher than 1024 example limit if needed.